tests/docker-compose.yml

```diff
@@ -60,4 +60,4 @@ services:
         TERM: xterm
     depends_on:
       - conjur
-      - conjur_cli
+      - conjur_cli
```

Comment:
I don't think this needed. Also you may as well add a newline to the end here and in tests.sh to make GH happy

Reply:
Done!